src/bugsnag.js

```diff
@@ -175,6 +225,181 @@
     }
   }
 
+  // Setup breadcrumbs for click events
+  function trackClicks() {
+    if(!getBreadcrumbSetting("autoBreadcrumbsClicks", true)) {
+      return;
+    }
+
+    window.addEventListener("click", function(event) {
```

Comment:
I think we only polyfill some minor behavior differences on different versions of `addEventListener`, so if it's not already there in a browser I don't know that we'd want to rely on bugsnag-js to polyfill it. As a third party it seems nicer/safer for us to not add new things to window if they're not there already, so I'd vote for us just skipping any automatic breadcrumbs that rely on events in browsers without addEventListener. (We could consider adding `attachEvent` support if a lot of people request it, I guess?)